mkdocs.yml

```diff
@@ -44,26 +39,35 @@ nav:
     - 'FirecREST': access/firecrest.md
     - 'JupyterLab': access/jupyterlab.md
     - 'VSCode': access/vscode.md
-  - 'Software':
-    - software/index.md
-    - 'Scientific Applications':
-      - software/sciapps/index.md
-      - 'CP2K': software/sciapps/cp2k.md
-      - 'GROMACS': software/sciapps/gromacs.md
-      - 'LAMMPS': software/sciapps/lammps.md
-      - 'NAMD': software/sciapps/namd.md
-      - 'Quantum ESPRESSO': software/sciapps/quantumespresso.md
-      - 'VASP': software/sciapps/vasp.md
+  - 'Environments':
+    - software-deployment/index.md
+    - 'Getting started in the terminal': guides/terminal.md
+    - 'uenv':
+      - software/uenv/index.md
+      - 'Configuration': software/uenv/configure.md
+      - 'Building uenv': software/uenv/build.md
+      - 'Deploying uenv': software/uenv/deploy.md
+      - 'Release notes': software/uenv/release-notes.md
+    - 'Container Engine': 
+      - software/container-engine/index.md
+      - 'Using container engine': software/container-engine/run.md
+      - 'Resource and hooks': software/container-engine/resource-hook.md
+      - 'EDF reference': software/container-engine/edf.md
+      - 'Known issues': software/container-engine/known-issue.md
+  - 'Building and Installing Software':
+    - build-install/index.md
     - 'Programming Environments':
       - software/prgenv/index.md
       - 'prgenv-gnu': software/prgenv/prgenv-gnu.md
       - 'prgenv-nvfortran': software/prgenv/prgenv-nvfortran.md
       - 'linalg': software/prgenv/linalg.md
       - 'julia': software/prgenv/julia.md
       - 'Cray modules (CPE)': software/prgenv/cpe.md
-    - 'Machine Learning':
-      - software/ml/index.md
-      - 'PyTorch': software/ml/pytorch.md
+    - 'How to build software':
+      - 'uenv': build-install/uenv.md
+    - 'Packaging and Deployment':
+      - 'Creating uenv': software/uenv/build.md
+      - 'Creating Containers with podman': build-install/containers.md
     - 'Communication Libraries':
```

Comment:
I think this belongs in `Applications and Frameworks` better than under `Bulding and Installing Software`. Maybe it's stretching the definition of `Applications and Frameworks` but at least my mental mapping for `Applications and Frameworks` is mostly "Things that are available on the system". `Communication Libraries` is to 99% libraries that we provide on the system in container hooks or uenvs rather than something generic that users have to install.